### PR TITLE
Set multiple picking to Mark As Todo

### PR DESCRIPTION
**Pull request by kenvac**
story: 13461

### CODE REVIEW

File: addons/udes_stock/wizard/mark_todo_stock_move.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        """Update draft pickings to waiting state"""
+        self.ensure_one()
+        draft_picking_ids = self.draft_picking_ids.ids
+        if not draft_picking_ids:
+            raise UserError(_("No draft transfer specified"))
+
+        self.draft_picking_ids.action_confirm()
+        return {
+            "type": "ir.actions.act_window",
+            "name": _("Transfers Mark as Todo"),
```
Comment (PeterC10) [inline addons/udes_stock/wizard/mark_todo_stock_move.py:65]: Super super nit: Suggest `Transfers Marked as Todo`

File: addons/udes_stock/wizard/mark_todo_stock_move.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
+from odoo import models, fields, api, _
+from odoo.exceptions import UserError
+
+
+class MarkTodoStockMove(models.TransientModel):
+    _name = "mark.todo.stock.move"
```
Comment (PeterC10) [inline addons/udes_stock/wizard/mark_todo_stock_move.py:6]: Suggestion: As this wizard deals with pickings directly maybe the name should be something like `stock.picking.mark.todo`, might be worth getting another opinion on this.

File: addons/udes_stock/wizard/mark_todo_stock_move.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    _description = "Update stock move in draft to Todo status"
```
Comment (PeterC10) [inline addons/udes_stock/wizard/mark_todo_stock_move.py:7]: Suggestion: Linked to the above, maybe something like `Mark Transfers as Todo Wizard`? Again worth getting another opinion. :)

**Review by PeterC10 — APPROVED** [review]

File: addons/udes_stock/wizard/stock_picking_mark_todo.py [hunk 4]
```diff
@@ -0,0 +1,70 @@
+from odoo import models, fields, api, _
+from odoo.exceptions import UserError
+
+
+class StockPickingMarkTodo(models.TransientModel):
+    _name = "stock.picking.mark.todo"
+    _description = "Mark Transfers as Todo Wizard"
+
+    @api.depends("non_draft_picking_ids")
+    def _compute_has_non_draft_pickings(self):
```
Comment (ws-mps) [inline addons/udes_stock/wizard/stock_picking_mark_todo.py:10]: Can we move this function after the fields definition please?

File: addons/udes_stock/wizard/stock_picking_mark_todo.py [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+    non_draft_picking_ids = fields.Many2many(
+        "stock.picking",
+        string="Non-Draft Pickings",
+        domain=[("state", "!=", "draft")],
+        readonly=True,
+    )
+
+    has_non_draft_pickings = fields.Boolean(
+        "Has Non-Draft Transfers?", compute="_compute_has_non_draft_pickings"
```
Comment (ws-mps) [inline addons/udes_stock/wizard/stock_picking_mark_todo.py:30]: Nit: Transfers or Pickings? We have used Pickings for the other two fields.

File: addons/udes_stock/wizard/stock_picking_mark_todo.py [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+        """Override default get to set values of draft and unaffected pickings
+
+        Args:
+            fields_list (list): list of fields
+
+        Returns:
+            dict: Returns fields dictionary with values
+        """
+        res = super().default_get(fields)
+        Picking = self.env["stock.picking"]
```
Comment (ws-mps) [inline addons/udes_stock/wizard/stock_picking_mark_todo.py:44]: Can we move this at the start of the function please?

File: addons/udes_stock/wizard/stock_picking_mark_todo.py [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+        """Update draft pickings to waiting state"""
+        self.ensure_one()
+        draft_picking_ids = self.draft_picking_ids.ids
+        if not draft_picking_ids:
+            raise UserError(_("No draft transfer specified"))
+
+        self.draft_picking_ids.action_confirm()
+        return {
+            "type": "ir.actions.act_window",
+            "name": _("Transfers Marked as Todo"),
```
Comment (ws-mps) [inline addons/udes_stock/wizard/stock_picking_mark_todo.py:65]: Same as above: Transfers vs Pickings

**Review by williamsajr — APPROVED** [review]

**Pull request merged**